### PR TITLE
Use traefik subdomains for clarity

### LINKED ISSUE #45: Explicitly set subdomain per application using traeffik
instead of using traeffik routing using the same domain, we should make every service use a dedicated subdomain. This will make the forwarding easier and avoid possible redirects on 404

### PR DESCRIPTION
**Pull request by minottic**
should close #45

### CODE REVIEW

**Comment by minottic** [comment]
this will have some conflicts with #47. TODO: make sure there is no mention of `catamel` after having fixed the conflicts

File: README.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ Files for running SciCat with docker-compose.
    ```sh
    docker-compose up -d
    ```
-3. SciCat will now be available on http://localhost. The Loopback API explorer of catamel is available at http://localhost/explorer/, the one for the search-api at http://localhost/panosc-explorer/.
+3. SciCat will now be available on http://localhost. 
+
+The Loopback API explorer of catamel is available at http://catamel.localhost/explorer/ and the APIs at http://catamel.localhost/api/.
```
Comment (LAShemilt) [inline README.md:18]: I think this should be "The Swagger UI for the backend is available at http://backend.localhost/explorer and the APIs at https://backend.localhost.api"

**Comment by minottic** [comment]
yes, that needs changing after #47 is merged (see comment). On the loopback API, we are still keeping that as this PR does not use the new be, but still the old one

**Review by fpotier — APPROVED** [review]

**Comment by minottic** [comment]
@LAShemilt your request has been fulfilled, could you please review again and eventually mark it as fulfilled? your request for change is blocking the merge otherwise. Thanks!

**Review by LAShemilt — APPROVED** [review]

**Pull request merged**